test/tests/utilities_internalTest.js

```diff
@@ -153,5 +153,18 @@ describe("Zotero.Utilities.Internal", function () {
 			assert.lengthOf(Object.keys(identifiers[0]), 1);
 			assert.propertyVal(identifiers[0], "PMID", id);
 		});
+		
+		it("should extract multiple old and new style arXivs", async function () {
+			var identifiers = ZUI.extractIdentifiers("0706.0044 0706.00441v1 hep-ex/9809001v1 math.GT/0309135");
```

Comment:
Should also have a test for a `arXiv:` prefix and some punctuation around them (e.g., a trailing period)